remove-other-tokens.py

```diff
@@ -7,25 +7,26 @@
 import logging
 
 __doc__ = """
-This scripts removes tokens of a given user except the one given by serial.
-#
+This script removes tokens of a given user except the one given by serial.
+
 This is a script that can be called by the privacyIDEA script handler. It
 is designed to run as a post event handler at /token/init. It will remove
 all tokens of the user but the token, which was just enrolled.
 
 The script can be configured to remove only tokens which share the type
 with the token that was just enrolled. It can act on all tokens or on the
-active ones only.
+active ones only. It can optionally restrict to those which have a
+specific tokeninfo (e.g. software tokens).
 
 It takes the arguments
 
-   remove-other-tokens.py --user <user> --realm <realm> --serial <token serial>
+   remove-other-user-tokens.py --user <user> --realm <realm> --serial <token serial>
```

Comment:
I do not get this. The PR displays two scripts. Is this one renamed?
So what is actually the name of the new file?

Reply:
done in 955c3d7